ethereum/contracts/pyth/PythGovernanceInstructions.sol

```diff
@@ -0,0 +1,153 @@
+// contracts/GovernanceStructs.sol
+// SPDX-License-Identifier: Apache 2
+
+pragma solidity ^0.8.0;
+
+import "../libraries/external/BytesLib.sol";
+import "./PythInternalStructs.sol";
+
+/**
+ * @dev `PythGovernanceInstructions` defines a set of structs and parsing functions
+ * for Pyth governance instructions.
+ */
+contract PythGovernanceInstructions {
+
+    using BytesLib for bytes;
+
+    enum GovernanceModule {
+        Core, // 0
+        Target, // 1
+        Attest // 2
+    }
```

Comment:
hmm... I'm a little confused by this. Perhaps this is essentially magic number I was asking for above?

Reply:
If so, it's still probably a good idea to have both a static magic number along with this enum. We do this in the pyth account data, and it's a good practice to prevent serialized messages from accidentally (or maliciously) being misinterpreted.